### PR TITLE
prepare new `kvdb-` release

### PR DESCRIPTION
**Pull request by ordian**
* kvdb 0.6
* kvdb-memorydb 0.6
* kvdb-rocksdb 0.8
* kvdb-web 0.6

### CODE REVIEW

File: kvdb-web/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.6.0] - 2020-05-01
 ### Breaking
 - Updated to the new `kvdb` interface. [#313](https://github.com/paritytech/parity-common/pull/313)
```
Comment (dvdplm) [inline kvdb-web/CHANGELOG.md:11]: Given a lot happened here, I wouldn't mind a bit more info on what happened here, totally optional ofc.

Comment (ordian) [reply kvdb-web/CHANGELOG.md:11]: you mean copy `kvdb` changelog changes to every impl's changelog (`kvdb-memorydb`, `kvdb-rocksdb`, `kvdb-web`)?

**Comment by dvdplm** [comment]
@niklasad1 @ascjones @sorpaas can we get a 👍 here please?

**Comment by ordian** [comment]
Published.

**Pull request merged**